### PR TITLE
Elyran Navy Corvette

### PR DESCRIPTION
**Pull request by shaky-jake**
Adds an Elyran Navy corvette to the Badlands and Elyran sectors. Has six player slots. Intended to effectively replace the outdated Elyran strike craft offship.
(This has been collecting dust in the "finishing touches" phase for three months because I completely forgot about it woops)

### CODE REVIEW

**Comment by shaky-jake** [comment]
!review

File: maps/away/ships/elyra/elyra_corvette/elyra_corvette.dm [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+		"nav_elyran_corvette_3",
+		"nav_elyran_corvette_4",
+		"nav_elyran_corvette_dock_port",
+		"nav_elyran_corvette_dock_starboard"
+	)
+
+	invisible_until_ghostrole_spawn = TRUE
+
+/obj/effect/overmap/visitable/ship/elyran_corvette/New()
+	designation = "[pick("Persepolis", "Damascus", "Medina", "Aemaq", "New Suez", "Bursa", "Republican", "Falcon", "Gelin", "Sphinx", "Takam", "Dandan", "Anqa", "Falak", "Uthra", "Djinn", "Roc", "Shadhavar", "Karkadann")]"
```
Comment (NonQueueingMatt) [inline maps/away/ships/elyra/elyra_corvette/elyra_corvette.dm:52]: A corvette shouldn't have the name of a planet, that's usually something reserved for capital ships. Pick less-important cities instead

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: maps/away/ships/elyra/elyra_corvette/elyra_corvette.dm (hunk 1 above)
Comment (shaky-jake) [reply maps/away/ships/elyra/elyra_corvette/elyra_corvette.dm:52]: These names were copied over from the existing elyran strike craft.

**Comment by shaky-jake** [comment]
Removed the names of planets from the name list

File: maps/away/ships/elyra/elyra_corvette/elyra_corvette.dm (hunk 1 above)
Comment (NonQueueingMatt) [reply maps/away/ships/elyra/elyra_corvette/elyra_corvette.dm]: Change those too then, please

**Review by NonQueueingMatt — APPROVED** [review]

File: maps/away/ships/elyra/elyra_strike_craft/elyra_strike_craft.dm [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /datum/map_template/ruin/away_site/elyran_strike_craft
```
Comment (DreamySkrell) [inline maps/away/ships/elyra/elyra_strike_craft/elyra_strike_craft.dm:1]: the old current elyra strike craft should be deleted, as the ship in this PR is effectively its superior replacement:
1. there shouldn't be two ships of the same faction, filling the same role
2. the old elyra strike craft is inferior and does not follow current mapping standards (it does not have atmos for example)

**Review by DreamySkrell — CHANGES_REQUESTED** [review]

**Comment by DreamySkrell** [comment]
As for the new ship itself, it's very nice, and from what I see it does follow all the current good mapping standards.

Only optional non-required request I guess, would be to use the new terminals instead of holoconsoles in places like the CIC/bridge/cockpit, cause IMO they fit there better.

**Comment by shaky-jake** [comment]
All requested changes addressed:
- Removed the old Elyran strike craft entirely
- Replaced holoconsoles with terminals in the shuttle and CIC

**Review by DreamySkrell — CHANGES_REQUESTED** [review]
still got holoconsoles near propulsion and ship guns
ping me when you change these to terminals and I'll approve

**Review by DreamySkrell — APPROVED** [review]

**Pull request merged**